docs/src/tutorials/datalabs-dask-cluster/getting-ready.md

```diff
@@ -0,0 +1,22 @@
+# 2. Getting ready
+
+The Dask workers run Python in a particular environment.
+If you do not specify a Conda environment for the Dask cluster,
+then it will run in a default environment.
+For some situations that will be fine,
+but in many situations it is less error-prone if the Dask workers
+and the notebook are using the same environments.
+This can be achieved by using the same Conda environment in the notebook and the cluster.
+
+This is the approach being taken in this tutorial.
+Therefore, before proceeding, ensure that you have:
+
+* a project
+* project storage
+* a notebook
```

Comment:
A JupyterLab notebook (should be fairly obvious though)

Reply:
Thanks, updated.